planet/cli/features.py

```diff
@@ -150,6 +150,36 @@ async def item_get(ctx, collection_id, feature_id, pretty):
         echo_json(feature, pretty)
 
 
+@command(items, name="delete")
+@click.argument("collection_id")
+@click.argument("feature_id", required=False)
+async def item_delete(ctx, collection_id, feature_id, pretty):
```

Comment:
```suggestion
async def item_delete(ctx, collection_id, feature_id):
```

#nit, `pretty` param is not needed for delete

Reply:
That's true, but currently `pretty` is an option that all commands get, so each function has to accept it. I changed this to `item_delete(ctx, collection_id, feature_id, *args, **kwargs)` to accept additional args without having to list out args we don't intend to use. Let me know what you think.

Reply:
nice, this looks good!